ui/test/test.ts

```diff
@@ -114,6 +114,19 @@ addOverlayListener('onPlayerChangedEvent', (e) => {
     } else if (detail.job === 'RPR' && detail.jobDetail) {
       jobInfo.innerText =
         `${detail.jobDetail.soul} | ${detail.jobDetail.shroud} | ${detail.jobDetail.enshroudMilliseconds} | ${detail.jobDetail.lemureShroud} | ${detail.jobDetail.voidShroud}`;
+    } else if (detail.job === 'PCT' && detail.jobDetail) {
+      jobInfo.innerText =
+        `${detail.jobDetail.palleteGauge} | ${detail.jobDetail.palleteGauge} | (${detail.jobDetail.creatureMotif}|${
+          detail.jobDetail.weaponMotif ? 'Weapon' : 'None'
+        }|${detail.jobDetail.landscapeMotif ? 'Landscape' : 'None'}) | (${
+          detail.jobDetail.depictions.join('|') || 'None'
+        }) | ${
```

Comment:
```suggestion
        `${detail.jobDetail.palleteGauge} | ${detail.jobDetail.paint} | (${detail.jobDetail.creatureMotif} | ${
          detail.jobDetail.weaponMotif ? 'Weapon' : 'None'
        } | ${detail.jobDetail.landscapeMotif ? 'Landscape' : 'None'}) | (${
          detail.jobDetail.depictions.join('+') || 'None'
        }) | ${
```

Reply:
Some minor formatting adjustments & prop change.
Also, purely stylistic suggestion so feel free to ignore - but perhaps consider joining the depictions with a `+` instead to distinguish from the field separator, if you think that makes sense?

Reply:
This looks make sense to me, thanks for the fixing.